setup.py

```diff
@@ -12,6 +12,7 @@
         ('share/' + package_name + '/resource', ['resource/RosGraph.ui']),
         ('share/' + package_name, ['package.xml']),
         ('share/' + package_name, ['plugin.xml']),
+        ('bin', ['bin/rqt_graph']),
```

Comment:
This is not installing an executable, but a Python script.
Thus, it's not runnable on Windows.

I'm not sure if there's a better alternative though.

Reply:
The entry point, which will result in an actual executable on Windows, needs to go to `bin` and the custom script to be invoked by `ros2 run` needs to be installed to `lib/<pkgname>`.

Reply:
I've reverted #42 (f3d688f) so the entry point goes to `bin` and installed the new script to `lib/rqt_graph` for `ros2 run` to find (a6b3654).

Reply:
This new approach looks better to me.
I would actually prefer to install an executable in both places, but I'm not sure if that's possible with setuptools.

Reply:
> I would actually prefer to install an executable in both places

Can you explain why that would be better? (Beside the desire to have it be the same.)

> I'm not sure if that's possible with setuptools.

I am not aware of a way to achieve that (excluding using custom setuptools commands for it).

Reply:
Installing scripts instead of executables is sometimes problematic, e.g.: you have to use special code in launch.
See https://github.com/ros2/demos/issues/374.

IMHO: We should never install scripts (neither in bin/ nor lib/<package_name>), only executables.

Reply:
I would argue that this is a shortcoming of `launch`. The command line tool `ros2 run` as well as launch in ROS 1 are working just fine with scripts. A package could also just provide a `ruby` / `perl` (or anything else) script and it needs to work.